DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -150,10 +159,10 @@ public void VerifyAdminUserCanAccessCourse_should_return_true_when_centreId_and_
         {
             // Given
             A.CallTo(() => courseDataService.GetCourseValidationDetails(A<int>._))
-                .Returns((2, 2));
+                .Returns((2, 2, false));
 
             // When
-            var result = courseService.VerifyAdminUserCanAccessCourse(1, 2, 2);
+            var result = courseService.VerifyAdminUserCanManageCourse(1, 2, 2);

```

Comment:
Test name `VerifyAdminUserCanAccessCourse_should_return_true_when_centreId_and_categoryId_match`
should change to match the new target of `VerifyAdminUserCanManageCourse`:
`VerifyAdminUserCanManageCourse_should_return_true_when_centreId_and_categoryId_match`